### PR TITLE
[PT-BR] [jsxImportSource.md, noUncheckedIndexedAccess.md]

### PR DESCRIPTION
**Pull request by gustavofabro**
@khaosdoctor @alvarocamillont @danilofuchs @orta

### CODE REVIEW

File: packages/tsconfig-reference/copy/pt/options/noUncheckedIndexedAccess.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+display: "noUncheckedIndexedAccess"
+oneline: "Adicionar undefined para um tipo quando acessado através de um índice"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/noUncheckedIndexedAccess.md]:
```suggestion
oneline: "Usa undefined para um tipo quando acessado através de um índice"
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]

**Comment by khaosdoctor** [comment]
Just a small typo change

File: packages/tsconfig-reference/copy/pt/options/noUncheckedIndexedAccess.md (hunk 1 above)
Comment (gustavofabro) [reply packages/tsconfig-reference/copy/pt/options/noUncheckedIndexedAccess.md]: Feito

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**